sfaira/models/embedding/output_layers.py

```diff
@@ -91,7 +91,116 @@ def call(self, inputs, **kwargs):
         mean_clip = tf.clip_by_value(mean, -bound, bound, "decoder_clip")
         var_clip = tf.clip_by_value(var, -bound, bound, "decoder_clip")
 
-        exp_mean = tf.exp(mean_clip + sf)
-        exp_var = tf.exp(var_clip)
+        invlinker_mean = tf.exp(mean_clip + sf)
+        invlinker_var = tf.exp(var_clip)
 
-        return [exp_mean, exp_var]
+        return [invlinker_mean, invlinker_var]
+
+
+class GaussianOutput(tf.keras.layers.Layer):
+    """
+    Gaussian output layer.
+
+    Size factor only makes sense if logged and data is positive and logged.
+    """
+
+    def __init__(
+            self,
+            original_dim=None,
+            name='neg_bin_output',
```

Comment:
@davidsebfischer did you on purpose not change the name of the output layer here?

Reply:
no, will change